cfgov/jinja2/v1/_includes/macros/notification.html

```diff
@@ -31,7 +31,7 @@
     {% set notification_icon_type  =  (' cf-notification_icon__' + type) if type %}
     {% set icon_style              =  (' cf-icon-' + icon_lookup[type] + '-round') if type %}
     {% set additional_classes = options.additional_classes  %}
-    <div class="cf-notification {{ notification_type ~ ' ' ~ additional_classes}} ">
+    <div class="cf-notification {{ notification_type ~ ' ' ~ (additional_classes or '') }} ">
```

Comment:
Should this be `<div class="cf-notification {{ notification_type ~ (' ' ~ additional_classes or '') }} ">`?